fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -77,6 +77,8 @@ public enum ErrorCode {
             "There is no such grant defined for user '%s' on host '%s'"),
     ERR_TABLEACCESS_DENIED_ERROR(1142, new byte[] {'4', '2', '0', '0', '0'},
             "%s command denied to user '%s'@'%s' for table '%s'"),
+    ERR_MV_ACCESS_DENIED_ERROR(1142, new byte[] {'4', '2', '0', '0', '0'},
```

Comment:
this error code same as previous 1142 better use different one.

Reply:
you are right, will fix it